pandas/tests/io/parser/parse_dates.py

```diff
@@ -365,7 +365,7 @@ def test_parse_tz_aware(self):
         assert stamp.minute == 39
         try:
             assert result.index.tz is pytz.utc
-        except AssertionError:  # hello Yaroslav
+        except AssertionError:
```

Comment:
actually why do we have an assertion error check  here at all?

Reply:
Not sure...that code is at least 2 years old.

I imagine that `pytz` compatibility has improved since then...

Reply:
can u try to remove this and see